### PR TITLE
[DDW-608] Implement new About dialog design

### PR DESCRIPTION
**Pull request by DeeJayElly**
This PR implements new About us dialog design with a close button included.

## Screenshots:
Screenshot 2019-04-01 11 26 46
Screenshot 2019-04-01 11 26 18

### Basics

- [ ] PR is updated to the most recent version of target branch (and there are no conflicts)
- [ ] PR has good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance tests are passing (`yarn run test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn run dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn run package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn run flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn run lint`)
- [ ] Text changes are proofread and approved (Jane Wild)
- [ ] There are no missing translations (running `yarn run manage:translations` produces no changes)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn run storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly documented and commented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-rendering
- [ ] Any code that only works in Electron is neatly separated from components

### Testing
- [ ] New feature / change is covered by acceptance tests
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature / change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review:
- [ ] Merge PR
- [ ] Delete source branch
- [ ] Move ticket to `done` on the Youtrack board

### CODE REVIEW

**Comment by DeeJayElly** [comment]
Done with the fixes. @nikolaglumac

**Comment by nikolaglumac** [comment]
@a-rukin can you please check the screenshot in the PR description? 🙏

**Comment by a-rukin** [comment]
- Small cardano logo is a bit more far (pls recheck values) 
- Hover effect for the close button seems to be too small (or cross icon is too big) 
- daedalus logo position is wrong
rest seems to be good

**Comment by DeeJayElly** [comment]
> * Small cardano logo is a bit more far (pls recheck values)
> * Hover effect for the close button seems to be too small (or cross icon is too big)
> * daedalus logo position is wrong
> 
> rest seems to be good

Done

File: source/renderer/app/components/static/About.scss [hunk 1]
```diff
@@ -1,11 +1,39 @@
 .container {
-  background-color: var(--theme-about-window-background-color);
   display: flex;
   flex-direction: column;
   height: 100%;
-  padding: 30px 30px 20px;
+  padding: 50px 80px 50px;
```
Comment (nikolaglumac) [inline source/renderer/app/components/static/About.scss]: @DeeJayElly please change to: `padding: 50px 80px;`

File: source/renderer/app/components/static/About.scss [hunk 2]
```diff
@@ -1,11 +1,39 @@
...
   user-select: text;
 
+  .closeButton {
+    cursor: pointer;
+    display: flex;
+    justify-content: flex-end;
+    top: 15px;
+    right: 15px;
+    position: absolute;
```
Comment (nikolaglumac) [inline source/renderer/app/components/static/About.scss]: @DeeJayElly please order CSS rules alphabetically 🙏

File: source/renderer/app/themes/daedalus/cardano.js [hunk 3]
```diff
@@ -400,15 +400,18 @@ export default {
...
-  '--theme-about-window-content-bottom-border-color': '#dfe4e8',
-  '--theme-about-window-copyright-color': '#5e6066',
+  '--theme-about-window-background-color': 'rgba(36, 62, 98, 0.96)',
+  '--theme-about-window-header-bottom-border-color': 'rgba(233, 244, 254, 0.3)',
+  '--theme-about-window-daedalus-icon-color': '#e9f4fe',
+  '--theme-about-window-cardano-icon-color': '#e9f4fe',
+  '--theme-about-window-title-varsion-color': '#e9f4fe',
+  '--theme-about-window-title-stroke-color': '#e9f4fe',
+  '--theme-about-window-content-color': '#fafbfc',
+  '--theme-about-window-content-text-color': 'rgba(250,251,252,0.7)',
```
Comment (nikolaglumac) [inline source/renderer/app/themes/daedalus/cardano.js]: @DeeJayElly please separate rgba values with space 🙏

File: source/renderer/app/themes/daedalus/dark-blue.js [hunk 4]
```diff
@@ -403,16 +403,18 @@ export default {
...
-  '--theme-about-window-background-color': '#263345',
+  '--theme-about-window-background-color': 'rgba(36, 62, 98, 0.96)',
   '--theme-about-window-header-bottom-border-color': 'rgba(233, 244, 254, 0.3)',
   '--theme-about-window-daedalus-icon-color': '#e9f4fe',
   '--theme-about-window-cardano-icon-color': '#e9f4fe',
   '--theme-about-window-title-varsion-color': '#e9f4fe',
   '--theme-about-window-title-stroke-color': '#e9f4fe',
-  '--theme-about-window-content-text-color': '#e9f4fe',
+  '--theme-about-window-content-color': '#fafbfc',
+  '--theme-about-window-content-text-color': 'rgba(250,251,252,0.7)',
```
Comment (nikolaglumac) [inline source/renderer/app/themes/daedalus/dark-blue.js]: @DeeJayElly please separate rgba values with space 🙏

File: source/renderer/app/themes/daedalus/light-blue.js [hunk 5]
```diff
@@ -399,15 +399,18 @@ export default {
...
-  '--theme-about-window-content-bottom-border-color': '#dfe4e8',
-  '--theme-about-window-copyright-color': '#5e6066',
+  '--theme-about-window-background-color': 'rgba(36, 62, 98, 0.96)',
+  '--theme-about-window-header-bottom-border-color': 'rgba(233, 244, 254, 0.3)',
+  '--theme-about-window-daedalus-icon-color': '#e9f4fe',
+  '--theme-about-window-cardano-icon-color': '#e9f4fe',
+  '--theme-about-window-title-varsion-color': '#e9f4fe',
+  '--theme-about-window-title-stroke-color': '#e9f4fe',
+  '--theme-about-window-content-color': '#fafbfc',
+  '--theme-about-window-content-text-color': 'rgba(250,251,252,0.7)',
```
Comment (nikolaglumac) [inline source/renderer/app/themes/daedalus/light-blue.js]: @DeeJayElly please separate rgba values with space 🙏

Comment (DeeJayElly) [reply source/renderer/app/themes/daedalus/light-blue.js]: @nikolaglumac Done

File: source/renderer/app/themes/daedalus/dark-blue.js (hunk 4 above)
Comment (DeeJayElly) [reply source/renderer/app/themes/daedalus/dark-blue.js]: @nikolaglumac Done

File: source/renderer/app/themes/daedalus/cardano.js (hunk 3 above)
Comment (DeeJayElly) [reply source/renderer/app/themes/daedalus/cardano.js]: @nikolaglumac Done

File: source/renderer/app/components/static/About.scss (hunk 2 above)
Comment (DeeJayElly) [reply source/renderer/app/components/static/About.scss]: @nikolaglumac Done

File: source/renderer/app/components/static/About.scss (hunk 1 above)
Comment (DeeJayElly) [reply source/renderer/app/components/static/About.scss]: @nikolaglumac Done

**Comment by DeeJayElly** [comment]
> @DeeJayElly good work! The responsive part works great now - the only issue which remains to be solved is the Close button which should be always visible but it gets out of the viewport in case you scroll:
> 
> button

Done

**Pull request merged**